### PR TITLE
RD-755 - System tests fixes

### PR DESCRIPTION
**Pull request by qooban**
I found that after https://github.com/cloudify-cosmo/cloudify-stage/pull/1209 merge there ([Job #214](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/214/pipeline/)) are two tests failing:
```
[2021-03-02T10:18:34.140Z]   │ ✖  tours_spec.js                            09:56        7        5        2        -        - │
[2021-03-02T10:18:34.141Z]   │ ✖  widgets/filter_spec.js                   09:30        3        1        2        -        - │
```

This PR fixes both. List of changes:
* CSS selectors fixed in tours JSON files and in Resource Filter widget test
* Make Resource Filter widget test to be aware of tooltip hiding Deployment Actions Menu options
* Extracted new Cypress custom command `searchInDeploymentsWidget` and reused it in Deployments widget and Resource Filter widget tests

Stage system tests results ([Job #216](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/216/pipeline)): 
```
[2021-03-02T12:51:44.508Z]        Spec                                              Tests  Passing  Failing  Pending  Skipped  
[2021-03-02T12:51:44.508Z]   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
[2021-03-02T12:51:44.508Z]   │ ✔  tours_spec.js                            00:52        7        7        -        -        - │
[2021-03-02T12:51:44.508Z]   ├────────────────────────────────────────────────────────────────────────────────────────────────┤
[2021-03-02T12:51:44.508Z]   │ ✔  widgets/deployments_spec.js              00:57       11       11        -        -        - │
[2021-03-02T12:51:44.508Z]   ├────────────────────────────────────────────────────────────────────────────────────────────────┤
[2021-03-02T12:51:44.508Z]   │ ✔  widgets/filter_spec.js                   00:25        3        3        -        -        - │
[2021-03-02T12:51:44.508Z]   └────────────────────────────────────────────────────────────────────────────────────────────────┘
[2021-03-02T12:51:44.508Z]     ✔  All specs passed!                        02:14       21       21        -        -        -  
```

### CODE REVIEW

**Review by Gelio — APPROVED** [review]
Just do be sure, did you verify that those pass now :smile: ?

**Comment by qooban** [comment]
@Gelio Yes, only locally, but as the issue was reproducible in the local environment, I believe that in the CI it will work, too 🙂

**Comment by Gelio** [comment]
Ok, great :slightly_smiling_face:

**Comment by qooban** [comment]
@Gelio Good that you raised concern. I ran the test for filter spec again and found actually that the test is "flaky" even with my change, so I'm not yet merging the PR, will try to fix it in a better way.

**Comment by qooban** [comment]
"Tooltip" provided by the `PopupMenu` component is a `Popup` component. In the Cypress tests it can happen that this `Popup`-tooltip hides part of the actual menu:
image

That was the problem here. I think that in the future we could start using standard tooltips in `PopupMenu` (provided by just adding `title` attribute). But as it requires general change in the cloudify-ui-components and we have here very specific case, only happening in tests, I don't want to go this way now. I'm putting it on my TODO list to create a ticket for that.

File: test/cypress/integration/widgets/filter_spec.js [hunk 1]
```diff
@@ -64,8 +64,9 @@ describe('Filter', () => {
 
             cy.get('.blueprintsWidget input[placeholder^=Search]').scrollIntoView().clear().type(blueprintName);
 
-            cy.contains('.deploymentsWidget .row', deploymentName).find('.green.checkmark');
-            cy.contains('.deploymentsWidget .row', deploymentName).find('.menuAction').click();
+            cy.searchInDeploymentsWidget(deploymentName);
+            // Triggering mouseout event just after the click to hide the tooltip
```
Comment (Gelio) [inline test/cypress/integration/widgets/filter_spec.js:68]: A TODO comment right here would be awesome if we are planning on fixing it later :slightly_smiling_face:

**Review by Gelio — APPROVED** [review]
An interesting case :smile: I'm wondering why the tooltip stays open after clicking the element - maybe we should somehow hack it so the tooltip is hidden of the popup is opened :thinking: 

I'm not sure if forcing using `title` won't limit us in the type of content we can show in the tooltip. I generally appreciate being able to render any `ReactNode` in the tooltip. With `title` we would be limited to only rendering strings

Anyhow, that's for later :slightly_smiling_face: Good that you caught it

I assume the tests are passing now :smile:

**Comment by qooban** [comment]
@Gelio Yes, tests are passing. See https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/216/pipeline/ - executed only on 3 relavant tests, all 3 are passing. 

(it's red due to coverage check failure, which is expected as for small set of tests coverage threshold is not reached)

File: test/cypress/integration/widgets/filter_spec.js (hunk 1 above)
Comment (qooban) [reply test/cypress/integration/widgets/filter_spec.js:68]: Yep, I can add it once I report related issue.

**Comment by Gelio** [comment]
Thanks for confirming :slightly_smiling_face:

**Pull request merged**